### DISCUSSION TITLE
Installation mit hm-mod-rpi-pcb im Supervised-Modus

### DISCUSSION THREAD

**Discussion by estegewr** [OpenCCU/OpenCCU#2298] [🇩🇪 Fragen&Antworten] [answered]
Ich habe gegoogelt, in (HA-)Foren recherchiert und GitHub durchsucht. Ich habe zwar ein paar Anhaltspunkte gefunden, aber nicht ausreichend, um bei meinem Problem weiter zu kommen. Aso versuche ich es hier, bin aber für jeden Hinweis für eine treffendere Quelle dankbar.

Ich versuche RaspberryMatic auf einem RPi4 mit hm-mod-rpi-pcb unter Home Assistant im Supervised-Modus zu nutzen. Hierzu habe ich die [Installationsanleitung](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant#using-homeassistant-supervised) befolgt. Dabei wurden keine Fehler angezeigt - zumindest keine für mich erkennbaren Fehler.

Beim Start von RaspberryMatic erhalte ich eine Log-Ausgabe, die nachfoglende aufgeführt ist. Der Ärger beginnt wohl beim Starten von multimacd bzw. der Initialisierung von /dev/eq3loop.

1. Wie kann ich zusätzliche (Debug-)Informationen generieren, um dem Problem einen Schritt neher zu kommen?
2. Gibt es Tipps&Tricks, die in dem Fall helfen könnten?
3. Ich hatte gelesen, dass es ab einer bestimmten Kernelversion ein paar Herausforderungen mit den Berechitungen gibt. Stimmt die Ausgabe bzgl. der Rechte, die im Supervisor-Log ausgegeben wird (siehe Log am Ende)?

Vielen Dank vorab für Hinweise und Ratschläge!

**Log-Ausgabe beim Starten von RaspberryMatic**
> Mounting /data as /usr/local (Home Assistant Add-On): OK
> Starting watchdog...
> Identifying host system: oci, OK
> Initializing RTC Clock: no hardware found
> Running sysctl: OK
> Checking for Factory Reset: not required
> Checking for Backup Restore: not required
> Initializing System: OK
> Setup ca-certificates: OK
> Starting logging: OK
> Init onboard LEDs: init, OK
> Starting irqbalance: OK
> Saving 256 bits of creditable seed for next boot
> Starting iptables: OK
> Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.4, OK
> Identifying Homematic RF-Hardware: ....HmRF: HM-MOD-RPI-PCB/GPIO@fe201000.serial, HmIP: HM-MOD-RPI-PCB/GPIO@fe201000.serial, OK
> Updating Homematic RF-Hardware: HM-MOD-RPI-PCB: 2.8.6, not necessary, OK
> Starting hs485dLoader: disabled
> Starting xinetd: OK
> Starting eq3configd: OK
> Starting lighttpd: OK
> Starting ser2net: disabled
> Starting ssdpd: OK
> Starting ha-proxy: OK
> Starting NUT services: disabled
> Initializing Third-Party Addons: OK
> Starting LGWFirmwareUpdate: ...OK
> Setting LAN Gateway keys: OK
> Starting hs485d: disabled
> Starting multimacd: ..........ERROR: couldn't setup /dev/eq3loop
> Starting rfd: ERROR: /dev/mmd_bidcos missing, no BidCos-RF hardware found
> Starting HMIPServer: ERROR: /dev/mmd_hmip missing
> Starting ReGaHss: .OK
> Starting CloudMatic: OK
> Starting NeoServer: disabled
> Starting Third-Party Addons: OK
> Starting crond: OK
> Setup onboard LEDs: booted, OK
> Finished Boot: 3.69.6.20230407 (raspmatic_oci_arm64)

**Log-Ausgabe des HA-Supervisors**

> 23-04-23 22:30:15 INFO (SyncWorker_0) [supervisor.docker.addon] Starting Docker add-on ghcr.io/jens-maus/raspberrymatic with version 3.69.6.20230407
> 23-04-23 22:30:29 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/eq3loop - None
> 23-04-23 22:30:29 INFO (MainThread) [supervisor.docker.addon] Added cgroup permissions 'c 239:0 rwm' for device /dev/eq3loop to addon_de838cd8_raspberrymatic

**Comment by Baxxy13** [comment]
Hier mal gucken:
#2119

**Reply by estegewr** [reply, marked as answer]
Das Problem scheint im OS_Agent v1.4.1 zu liegen. In der von @Baxxy13 referenzierten Diskussion wurde ein Downgrade auf Version 1.2.2 empfohlen. Es funktioniert nun aber auch ein Upgrade auf die aktuelle Version 1.5.1!
